### PR TITLE
Add: Windows DLL version info header, this solves issue …

### PR DESCRIPTION
**Pull request by Blonder**
https://github.com/zeux/pugixml/issues/294

### CODE REVIEW

File: src/win32/pugixml.rc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#include <winver.h>
+//#include "../pugixml.hpp"
+// TODO these defines should be moved to ../pugixml.hpp
```
Comment (zeux) [inline src/win32/pugixml.rc]: Yeah I doubt the resource compiler can ever parse the full header. I think it's fine to leave the version numbers duplicated here, they are already spread across many many files and my usual process is to search for all occurrences of 1.9 and change them to 1.10, so this works :)

Comment (Blonder) [reply src/win32/pugixml.rc]: 20190918_pugixml_windows_version_number

Comment (Blonder) [reply src/win32/pugixml.rc]: We could add 
`# define PUGIXML_VERSION_MAJOR 1`
`# define PUGIXML_VERSION_MINOR 10`
`# define PUGIXML_VERSION_PATCH 0`
`# define PUGIXML_VERSION_NUMBER "1.10.0\0"`
to the `PUGIXML_VERSION` section and
`#ifndef RC_INVOKED` in the `pugixml.hpp` header for the rest of the header.

Comment (zeux) [reply src/win32/pugixml.rc]: Ah I guess. I think it's fine like this.

**Pull request merged**